redash/query_runner/uptycs.py

```diff
@@ -0,0 +1,136 @@
+from redash.query_runner import *
+from redash.utils import json_dumps
+
+import json
+import jwt
+import datetime
+import requests
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class Uptycs(BaseSQLQueryRunner):
+    noop_query = "SELECT 1"
+
+    @classmethod
+    def configuration_schema(cls):
+        return {
+            "type": "object",
+            "properties": {
+                "url": {
+                    "type": "string"
+                },
+                "customer_id": {
+                    "type": "string"
+                },
+                "key": {
+                    "type": "string"
+                },
+                "SSL": {
```

Comment:
Should be lowered cases (like `ssl`) and `verify_ssl` is probably more descriptive.

Reply:
Renamed to verify_ssl